tools/authors/addbio.php

```diff
@@ -114,11 +117,19 @@
 <form name="addform" action="addbio.php" method="POST">
 <input type="hidden" name="author_id" value="<?php echo $author_id; ?>">
 <?php
-if (isset($bio_id))
+if (isset($bio_id)) {
     echo '<input type="hidden" name="bio_id" value="' . $bio_id . '">';
+    echo '<input type="hidden" name="bio_format" value="' . $bio_format . '">';
+}
+
+if (!isset($bio_id) || $bio_format === 'markdown') {
+    $comment_format_text = "<a href=''>" . _("Markdown") . "</a>";
```

Comment:
Let's include the link here: https://www.pgdp.net/wiki/Markdown

Reply:
Also, we should not wrap either `HTML` or `Markdown` in `_()` since those are proper nouns and should not be translated differently, so this can simply be:
```
    $comment_format_text = "<a href='https://www.pgdp.net/wiki/Markdown'>Markdown</a>";
```

Reply:
whoops, I left the href to grab the url later and insert. /facepalm.